resume.py

```diff
@@ -165,7 +165,59 @@ def write_pdf(html: str, prefix: str = "resume", chrome: str = "") -> None:
             logging.info(exc)
 
 
-if __name__ == "__main__":
+def write_pdf_win32(prefix: str = "resume", chrome: str = "") -> None:
+    """
+    Write html to prefix.pdf
+
+    Sample command which can be run from PowerShell:
+    & 'C:\Program Files (x86)\Google\Chrome\Application\chrome.exe' --headless --disable-gpu --run-all-compositor-stages-before-draw 
+    --print-to-pdf-no-header --virtual-time-budget=5000 --no-margins --print-to-pdf=C:\work\code\resume.md\resume.pdf C:\work\code\resume.md\resume.html
+    """
+    chrome = chrome or guess_chrome_path()
+
+    cwd = os.getcwd()
+    input_html = os.path.join(cwd, f"{prefix}.html")
+    output_pdf = os.path.join(cwd, f"{prefix}.pdf")
+    
+    options = [
+        "--headless",
+        "--print-to-pdf-no-header",
+        "--run-all-compositor-stages-before-draw",
+        "--virtual-time-budget=5000",
```

Comment:
Could you please explain these new options? What do they change? Why are they necessary? Should they be applied on other platforms?

Reply:
`--run-all-compositor-stages-before-draw` and `--virtual-time-budget` are optional and used to allow time for chrome to render all the UI components. Refer this: https://stackoverflow.com/questions/61914997/how-to-set-headless-chrome-wait-until-the-page-is-fully-loaded-before-printing 
We can potentially remove this, since we do not have much UI elements. I had added them for safety. But we should be good without them.